TestHelpers.Tests/MockFileCopyTests.cs

```diff
@@ -120,7 +131,7 @@ public void MockFile_Copy_ShouldThrowNotSupportedExceptionWhenSourceFileNameCont
         }
 
         [Test]
-        public void MockFile_Copy_ShouldThrowNotSupportedExceptionWhenSourcePathContainsInvalidChars_Message()
+        public void MockFile_Copy_ShouldThrowArgumentExceptionWhenSourcePathContainsInvalidChars_Message()
         {
             if (XFS.IsUnixPlatform())
```

Comment:
While not necessarily part of the PR, we're touching the line right above it. It'd be nice to push forward and get rid of these if() checks and use the attribute.